core/templates/third-party-imports/wave-surfer.scripts.ts

```diff
@@ -0,0 +1,2 @@
+window.WaveSurfer = require(
+  '../../../third_party/static/wave-surfer-js-2.2.1/wavesurfer.min.js');
```

Comment:
Relative imports are not allowed. The lint check for this doesn't work properly but #8870 should fix it. But meanwhile, can you please correct this instance? Thanks!

Reply:
@kevintab95 Fixed this.